metadata-ingestion/src/datahub/ingestion/source/dbt/dbt_common.py

```diff
@@ -394,9 +398,10 @@ def get_urn(
         target_platform: str,
         env: str,
         data_platform_instance: Optional[str],
+        convert_urns_to_lowercase: bool,
     ) -> str:
         db_fqn = self.get_db_fqn()
-        if target_platform != DBT_PLATFORM:
+        if (target_platform != DBT_PLATFORM) and convert_urns_to_lowercase:
```

Comment:
This part concerns me because it won't assure backwards compatibility. If `convert_urns_to_lowercase=False` (default), then `db_fqn` won't get lowercased. It may cause problems in setups where DBT ingestion is already running. Possible mismatch to source platform nodes? 🤔

Let me know what you think and if we can do it differently. The other parts of the code keep backwards compatibility when `convert_urns_to_lowercase=False`

Reply:
I need to check this in more detail, but I thought this was supposed to lowercase all outbound lineage edges by default. Basically dbt nodes retain their original casing, but their lineage to the snowflake/bigquery/etc tables would always be lowercased

Reply:
To make sure I understand, what was the issue you were seeing that necessitated this change? Was it that the overall lineage didn't match up, or was it related to column-level lineage?